src/customers/repository/customer.repository.ts

```diff
@@ -1,7 +1,8 @@
-import { Injectable } from "@nestjs/common";
+import { Injectable, NotFoundException } from "@nestjs/common";
 import { InjectRepository } from "@nestjs/typeorm";
 import { Repository } from "typeorm";
 import { Customer } from "../domain/customer.entity";
+import { UUID } from "crypto";
```

Comment:
**suggestion(non-blocking):** careful about `import` statement that you are not using, you should remove them

Reply:
![image](https://github.com/user-attachments/assets/524d0d5b-ebb3-489f-86fa-52a17f2abca2)

Enabled auto-organize imports on code save. Also tried with `npx eslint . --fix`, but it didn't work.

If I remember correctly, we can globally organize imports using Vite, before building the application for deployment, though.

Reply:
Just be careful with your code for now